### PR TITLE
L2D5A1

### PR DESCRIPTION
**Pull request by evyryan**

### CODE REVIEW

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 1]
```diff
@@ -102,8 +103,8 @@
       <lb ed="#V" n="33"/>amor: et non superbia. 
    </p>
                <p xml:id="kzz7yh-f17284-d1e178">
-                  <lb ed="#V" n="34"/>Contra Ecclu. 10 Initium omnis peccati est superbia 
-      <lb ed="#V" n="35"/>Sed primum peccatum luciferi: fuit initiu 
+                  <lb ed="#V" n="34"/>Contra Ecclu <!-- TODO abbreviation-->. 10 Initium omnis peccati est superbia
```
Comment (RobbieBuhite) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:106]:
```suggestion
                  <lb ed="#V" n="34"/>Contra Eccl. 10 Initium omnis peccati est superbia 
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 2]
```diff
@@ -305,13 +306,14 @@
             </div>
             <div xml:id="kzz7yh-f17284-Dd1e532">
                <head xml:id="kzz7yh-f17284-Hd1e534">Quaestio 2</head>
+               <head type="question-title">Utrum peccatum angelorum qui lucifero adheferunt  fuerit superbia vel pusillanimitas</head>
```
Comment (RobbieBuhite) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:309]:
```suggestion
               <head type="question-title">Utrum peccatum angelorum qui lucifero adheserunt fuerit superbia vel pusillanimitas</head>
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 3]
```diff
@@ -305,13 +306,14 @@
...
                <p xml:id="kzz7yh-f17284-d1e536">
       ¶ Questio. II. 
-      <lb ed="#V" n="24"/>Ecundo queritur vtrum peccatum 
-      ange<lb ed="#V" n="25" break="no"/>iorum qui lucifero adheserunt 
+      <lb ed="#V" n="24"/>SEcundo queritur vtrum peccatum 
+      ange<lb ed="#V" n="25" break="no"/>lorum qui lucifero adheferunt <!--TODO: abbreviation? (Question header as well)-->
```
Comment (RobbieBuhite) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:313]:
```suggestion
      ange<lb ed="#V" n="25" break="no"/>lorum qui lucifero adheserunt <!--TODO: Question header-->
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 4]
```diff
@@ -305,13 +306,14 @@
...
       fue<lb ed="#V" n="26" break="no"/>rit superbia vel pusillanimitas. Et videtur quod 
-      <lb ed="#V" n="27"/>fuerit pusillanimitas. Quia potius velle subeem 
-      mi<lb ed="#V" n="28" break="no"/>nori quam maiori pusillanimitas videtur. Sed angeli qui 
+      <lb ed="#V" n="27"/>fuerit pusillanimitas. Quia potius velle subeem <!-- TODO Idk this latin word--> mi
```
Comment (RobbieBuhite) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:315]:
```suggestion
      <lb ed="#V" n="27"/>fuerit pusillanimitas. Quia potius velle subesse
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 5]
```diff
@@ -305,13 +306,14 @@
...
+      <lb ed="#V" n="28"/>nori quam maiori pusillanimitas videtur. Sed angeli qui
```
Comment (RobbieBuhite) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:316]:
```suggestion
      mi<lb ed="#V" n="28" break="no"/>nori quam maiori pusillanimitas videtur. Sed angeli qui 
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 6]
```diff
@@ -335,7 +337,7 @@
       er<lb ed="#V" n="43" break="no"/>go peccato pusillanimitatis peccauerunt. 
    </p>
                <p xml:id="kzz7yh-f17284-d1e588">
-                  <lb ed="#V" n="44"/>Contra Hug iet libro de sacra. par. . c. 33 Ex illis qui 
+                  <lb ed="#V" n="44"/>Contra Hug.i.libro.de sacra. par. . c. 33 <!--TODO abbreviation--> Ex illis qui
```
Comment (RobbieBuhite) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:340]:
```suggestion
                  <lb ed="#V" n="44"/>Contra Hug.i.libro.de sacra. par. 5. c. 33. Ex illis qui 
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 7]
```diff
@@ -401,12 +403,13 @@
...
                <head xml:id="kzz7yh-f17284-Hd1e710">Quaestio 3</head>
+               <head type="question-title">Utrum culpa luciferi fuerit culpe aliorum aliquo modo causa</head>
                <p xml:id="kzz7yh-f17284-d1e712">
       Quaesio III. 
-      <lb ed="#V" n="88"/>Tertio queritur vtrum culpa 
+      <lb ed="#V" n="88"/>TErtio queritur vtrum culpa 
       luci<lb ed="#V" n="89" break="no"/>feri fuerit 
       cul<lb ed="#V" n="90" break="no"/>pe aliorum aliquo modo causa. Et videtur quod non. Quia 
-      sta<lb ed="#V" n="91" break="no"/>tim cum lucifer peccauit deformatas est: alii autem 
+      sta<lb ed="#V" n="91" break="no"/>tim cum lucifer peccauit deformatus est: alii autem <!-- TODO: Auteum or aut?-->
```
Comment (RobbieBuhite) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:412]:
```suggestion
      sta<lb ed="#V" n="91" break="no"/>tim cum lucifer peccauit deformatus est: alii autem
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 8]
```diff
@@ -433,8 +436,8 @@
       <lb ed="#V" n="107"/>causa peccati aliorum neque occasio. 
    </p>
                <p xml:id="kzz7yh-f17284-d1e767">
-                  <lb ed="#V" n="108"/>Contra Apoc. 2 dicitur quod draco cauda sua trahebat 
-      ter<lb ed="#V" n="109" break="no"/>tia partem steliarum celi et misit eas in terram. Sed 
+                  <lb ed="#V" n="108"/>Contra Apoc. i2.dicitur quod draco cauda sua trahebat 
+      ter<lb ed="#V" n="109" break="no"/>tia partem stellarum celi et misit eas in terram. Sed
```
Comment (RobbieBuhite) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:440]:
```suggestion
      ter<lb ed="#V" n="109" break="no"/>tiam partem stellarum celi et misit eas in terram. Sed 
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 9]
```diff
@@ -447,8 +450,8 @@
       <lb ed="#V" n="116"/>lucifer aliis suggerendo malitiam fuit occasio peccati eorum. 
    </p>
                <p xml:id="kzz7yh-f17284-d1e791">
-                  <lb ed="#V" n="117"/>Respondeo quod peccatum luciferi nonfuit causa propise 
-      <lb ed="#V" n="118"/>ioquendo peccati aliorum quia peccatum 
+                  <lb ed="#V" n="117"/>Respondeo quod peccatum luciferi non fuit causa propise
```
Comment (RobbieBuhite) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:453]:
```suggestion
                  <lb ed="#V" n="117"/>Respondeo quod peccatum luciferi non fuit causa propositione 
```

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 10]
```diff
@@ -335,7 +337,7 @@
       er<lb ed="#V" n="43" break="no"/>go peccato pusillanimitatis peccauerunt. 
    </p>
                <p xml:id="kzz7yh-f17284-d1e588">
-                  <lb ed="#V" n="44"/>Contra Hug iet libro de sacra. par. . c. 33 Ex illis qui 
+                  <lb ed="#V" n="44"/>Contra Hug.i.libro.de sacra. par. 5. c. 33. Ex illis qui
```
Comment (jeffreycwitt) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:340]:
```suggestion
                  <lb ed="#V" n="44"/>Contra Hug. i. libro de sacra. par. 5. c. 33. Ex illis qui 
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 11]
```diff
@@ -305,12 +306,13 @@
             </div>
             <div xml:id="kzz7yh-f17284-Dd1e532">
                <head xml:id="kzz7yh-f17284-Hd1e534">Quaestio 2</head>
+               <head type="question-title">Utrum peccatum angelorum qui lucifero adheserunt fuerit superbia vel pusillanimitas</head>
                <p xml:id="kzz7yh-f17284-d1e536">
       ¶ Questio. II. 
-      <lb ed="#V" n="24"/>Ecundo queritur vtrum peccatum 
-      ange<lb ed="#V" n="25" break="no"/>iorum qui lucifero adheserunt 
+      <lb ed="#V" n="24"/>SEcundo queritur vtrum peccatum 
+      ange<lb ed="#V" n="25" break="no"/>lorum qui lucifero adheserunt <!--TODO: Question header-->
```
Comment (jeffreycwitt) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:313]:
```suggestion
      ange<lb ed="#V" n="25" break="no"/>lorum qui lucifero adheserunt 
```

File: kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml [hunk 12]
```diff
@@ -433,8 +436,8 @@
       <lb ed="#V" n="107"/>causa peccati aliorum neque occasio. 
    </p>
                <p xml:id="kzz7yh-f17284-d1e767">
-                  <lb ed="#V" n="108"/>Contra Apoc. 2 dicitur quod draco cauda sua trahebat 
-      ter<lb ed="#V" n="109" break="no"/>tia partem steliarum celi et misit eas in terram. Sed 
+                  <lb ed="#V" n="108"/>Contra Apoc. i2.dicitur quod draco cauda sua trahebat 
+      ter<lb ed="#V" n="109" break="no"/>tiam partem stellarum celi et misit eas in terram. Sed
```
Comment (jeffreycwitt) [inline kzz7yh-f17284/cod-ve09v2_kzz7yh-f17284.xml:440]:
```suggestion
                  <lb ed="#V" n="108"/>Contra Apoc. 12. dicitur quod draco cauda sua trahebat 
      ter<lb ed="#V" n="109" break="no"/>tiam partem stellarum celi et misit eas in terram. Sed 
```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]
Looking good. Just a few more changes to make.

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**